r/tools/nixlibs.R

```diff
@@ -295,32 +295,32 @@ determine_binary_from_stderr <- function(errs) {
   if (is.null(attr(errs, "status"))) {
     # There was no error in compiling: so we found libcurl and OpenSSL >= 1.1,
     # openssl is < 3.0
-    lg("Found libcurl and OpenSSL >= 1.1")
+    lg("Found headers for libcurl and OpenSSL >= 1.1")
```

Comment:
Any reason we're adding in "headers for" here + a bunch below? I get that it's most accurate, though I think for the largest proportion of our audience installing arrow mentioning "headers" here is more confusing than it is helpful. Do we contrast that with anything in these logs?

Reply:
If it's for us, I would love to bump those into comments above each of these or put those into a verbose mode, and maintain the more approachable and shorter messages we've had here

Reply:
I am ambivalent on this as the issue of seeing 'found libcurl' followed by 'failed to find libcurl' was cleared up by generalizing the second message but this version is also more accurate so :shrug:

Reply:
I reverted the changes to the test compilation messages